projects/storefrontlib/shared/components/media/media.component.ts

```diff
@@ -7,7 +7,6 @@ import {
   OnChanges,
   Output,
 } from '@angular/core';
-import { Image, ImageGroup } from '@spartacus/core';
```

Comment:
Why change this file? This doesn't seem to fit the scope of the PR.
Let's discuss.

Reply:
Don't know why this is here. I didn't make any changes to this file. Maybe during a /develop branch merge it changed these files